tests/tool_use/utils.py

```diff
@@ -187,7 +187,11 @@ def ensure_system_prompt(messages: list[dict[str, Any]],
                 "unit": {
                     "type": "string",
                     "description": "The unit to fetch the temperature in",
-                    "enum": ["celsius", "fahrenheit"]
+                    "oneOf": [{
+                        "const": "celsius"
+                    }, {
+                        "const": "fahrenheit"
+                    }]
```

Comment:
enum should work now

https://github.com/vllm-project/vllm/pull/15594

https://github.com/vllm-project/vllm/pull/15878